scripts/partner_batch_imports.py

```diff
@@ -183,8 +191,8 @@ def batch_import(path, batch, batch_size=5000):
 
                 try:
                     book_items.append(csv_to_ol_json_item(line))
-                except UnicodeDecodeError:
-                    pass
+                except AssertionError as e:
```

Comment:
I think this **will** appear as a `UnicodeDecodeError` and not an `AssertionError`